<launch>
	<arg name="port_pixhawk" />
	
	<node pkg="obstacle_avoidance" type="rploa.py" name="rploa" output="screen">
		<param name="port"         type="string" value="$(arg port_pixhawk)"/>  
		<param name="withdraw_dis" value="0.3" />
		<param name="disregard_dis" value="0.4" />
		<param name="min_valid_range" value="0.2" />
	</node>
</launch>
